src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -45,25 +44,26 @@ class Ledger(vm: VM) extends Logger {
     * This function runs transaction
     *
     * @param block
+    * @param blockchain
     * @param storages
-    * @param stateStorage
+    * @param validators
     */
   private def executeBlockTransactions(
     block: Block,
     blockchain: Blockchain,
     storages: BlockchainStorages,
-    stateStorage: NodeStorage):
+    validators: Validators):
   Either[BlockExecutionError, BlockResult] = {
     val parentStateRoot = blockchain.getBlockHeaderByHash(block.header.parentHash).map(_.stateRoot)
-    val initialWorld = InMemoryWorldStateProxy(storages, stateStorage, parentStateRoot)
+    val initialWorld = InMemoryWorldStateProxy(storages, storages.nodeStorage, parentStateRoot)
 
     val config = EvmConfig.forBlock(block.header.number)
 
-    log.debug(s"About to execute txs from block ${block.header}")
-    val blockTxsExecResult = executeTransactions(block.body.transactionList, initialWorld, block.header, config)
+    log.debug(s"About to execute ${block.body.transactionList.size} txs from block ${Hex.toHexString(block.header.hash.toArray)}")
```

Comment:
As it seems we will print this in several places, maybe it's a good idea to create `toHexString` method within the block, what do you think?